PF/Searching/TestingOfSearching.java

```diff
@@ -0,0 +1,22 @@
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+
+public class TestingOfSearching {
+
+	 @Test
+	    public void linearSearchingTest() {
```

Comment:
test method name should be more meaningful testLinearSearchingShouldSearchElementWithLinearSearch
test + method name + method description